bin.js

```diff
@@ -11,18 +11,17 @@ const CONSTANTS = require('./lib/constants')
 args
   .option(['c', 'colorize'], 'Force adding color sequences to the output')
   .option(['f', 'crlf'], 'Append CRLF instead of LF to formatted lines')
-  .option(['d', 'dateFormat'], 'A format string to govern display of dates. When not set to the default value, `--translateTime` is implied', CONSTANTS.DATE_FORMAT)
   .option(['e', 'errorProps'], 'Comma separated list of properties on error objects to show (`*` for all properties)', '')
   .option(['l', 'levelFirst'], 'Display the log level as the first output field')
   .option(['k', 'errorLikeObjectKeys'], 'Define which keys contain error objects (`-k err,error`)', 'err,error')
   .option(['m', 'messageKey'], 'Highlight the message under the specified key', CONSTANTS.MESSAGE_KEY)
-  .option(['n', 'localTime'], 'Display timestamps according to system timezone')
-  .option(['t', 'translateTime'], 'Convert Epoch timestamps to ISO format')
+  .option(['t', 'translateTime'], 'Display epoch timestamps as UTC ISO format, also accept a formatString to alter the output (default ISO 8601)')
```

Comment:
"Display epoch timestamps as UTC ISO format or according to an optional format string (default ISO 8601)"